### PR TITLE
[SELS-TASK]Created the migrations and models for user and cateogries

### PR DESCRIPTION
**Pull request by jarimesina**
[Commands]
	-Use php laravel serve to test if working
 [Pre-conditions]
	-none.
[Expected Output]
 	-no visible functionality so expect to see the laravel page
[Notes]
	-only files were created

### CODE REVIEW

File: database/migrations/2019_06_26_045023_create_admin_table.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('admin', function (Blueprint $table) {
+            $table->bigIncrements('id');
```
Comment (jeremiahC) [inline database/migrations/2019_06_26_045023_create_admin_table.php]: - where are the other columns for admin table?
 - please make admin plural

File: database/migrations/2019_06_26_045134_create_categories_table.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('categories', function (Blueprint $table) {
+            $table->bigIncrements('id');
```
Comment (jeremiahC) [inline database/migrations/2019_06_26_045134_create_categories_table.php:17]: - where are the other columns

File: app/admin.php [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class admin extends Model
```
Comment (jeremiahC) [inline app/admin.php]: class names are supposed to be CamelCase and also file name

File: app/category.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class category extends Model
+{
+    //
+}
```
Comment (jeremiahC) [inline app/category.php]: class names are supposed to be CamelCase and also file name

File: app/admin.php (hunk 3 above)
Comment (jarimesina) [reply app/admin.php]: admin.php was deleted

File: app/category.php (hunk 4 above)
Comment (jarimesina) [reply app/category.php]: category.php was deleted

File: database/migrations/2019_06_26_045023_create_admin_table.php (hunk 1 above)
Comment (jarimesina) [reply database/migrations/2019_06_26_045023_create_admin_table.php]: admin file was deleted

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**